backend/README.md

```diff
@@ -436,21 +393,21 @@ In Tracim, only some users can access to some information, this is also true in
 the Tracim REST API. you can check the [roles documentation](doc/roles.md) to check
 what a specific user can do.
 
-# Known Issues
+## Known Issues
 
 see [here](doc/known_issues.md)
 
-# Documentation
-- [apache.md](apache.md): Using Apache as a proxy for Tracim
-- [api.md](api.md): Using the Tracim API
-- [cli.md](cli.md): Controlling Tracim from the Command Line Using `tracimcli`
-- [database.md](database.md): Handling the Database (deprecated)
-- [devtools.md](devtools.md): Miscellaneous Information About the Developer Tools
-- [hello_world_plugin.py](hello_world_plugin.py): an hello world plugin
-- [i18n.md](i18n.md): Translating the Backend
-- [known_issues.md](known_issues.md): Known issues with the Tracim Backend
-- [migrate_from_v1.md](migrate_from_v1.md): Migrate from Tracim v1
-- [migration.md](migration.md): Performing Migrations
-- [roles.md](roles.md): Roles in Tracim
-- [setting.md](setting.md): Setting up Tracim
-- [webdav.md](webdav.md): Using WebDAV on Various Operating Systems
+## Other Documentation
+- [apache.md](doc/apache.md): Using Apache as a proxy for Tracim
+- [api.md](doc/api.md): Using the Tracim API
+- [cli.md](doc/cli.md): Controlling Tracim from the Command Line Using `tracimcli`
+- [database.md](doc/database.md): Handling the Database (deprecated)
+- [devtools.md](doc/devtools.md): Miscellaneous Information About the Developer Tools
+- [hello_world_plugin.py](doc/hello_world_plugin.py): an hello world plugin
+- [i18n.md](doc/i18n.md): Translating the Backend
+- [known_issues.md](doc/known_issues.md): Known issues with the Tracim Backend
+- [migrate_from_v1.md](-oc/migrate_from_v1.md): Migrate from Tracim v1
+- [migration.md](doc/migration.md): Performing Migrations
+- [roles.md](doc/roles.md): Roles in Tracim
+- [setting.md](doc/setting.md): Setting up Tracim
+- [webdav.md](doc/webdav.md): Using WebDAV on Various Operating Systems
```

Comment:
Missing documentation here (ex: env_settings.md).

Do we need to list all documentation here ? 
- If yes we need to update the list
- if no, maybe just add some information like this : `you can find more documentation [here](doc/) about apache configuration, api documentation, setings, etc.`

Reply:
I updated the links as they didn't include the `doc/` directory (thus their links were broken). Listing all docs here is here for the readers but we can easily forget to add a new one (as you noted)… I'll apply your suggestion.

Reply:
Great.